src/components/AddCard.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {useContext} from "react"
+import {AddCardProps} from "../interface/AddCardProps.ts";
+import CardContext from "../context/CardContext.tsx";
+import {CardType} from "../type/CardType.ts";
+import {useStepper} from "../context/StepperContext.tsx";
+
+const AddCard: React.FC<AddCardProps> = ({cardName, cardNumber, name, expireDate}) => {
+    const {addCard} = useContext(CardContext)
+    const {setCurrentStep} = useStepper()
+
+    const handleClick = () => {
+        const {month, year} = expireDate
+        const cardExpireDate = `${month} / ${year}`
+        const newCard: CardType = {cardName, cardNumber, name, cardExpireDate}
+        addCard(newCard)
+        setCurrentStep("AddCardComplete")
+    }
+
+    return (
+        <div className="button-box">
+            <span className="button-text" onClick={handleClick}>다음</span>
+        </div>
+    )
+}
```

Comment:
c:
- button 태그 대신 div로 구현된 버튼 컴포넌트들이 많이 보이네요. div는 비대화형 요소라 tab으로 선택하려면 tabIndex를 설정해야하고 disabled 속성이 없는 등 버튼 컴포넌트에 적합하지 않아 변경하는게 좋을 것 같습니다.
- 이 컴포넌트도 미션 요구사항에 맞게 bottom up 방식으로 리팩터링 할 수 있을 것 같아요.

```suggestion
const AddCard: React.FC<AddCardProps> = ({cardName, cardNumber, name, expireDate}) => {
    const {addCard} = useContext(CardContext)
    const {setCurrentStep} = useStepper()

    const handleClick = () => {
        const {month, year} = expireDate
        const cardExpireDate = `${month} / ${year}`
        const newCard: CardType = {cardName, cardNumber, name, cardExpireDate}
        addCard(newCard)
        setCurrentStep("AddCardComplete")
    }

    return (
        <Button onClick={handleClick}>
            <Text>다음</Text>
        </Button>
    )
}
```

Reply:
Button 추가 하였습니다.